recipe/meta.yaml

```diff
@@ -77,7 +77,7 @@ outputs:
       run:
         - python
         - {{ pin_subpackage("jupyterhub-base", exact=True) }}
-        - configurable-http-proxy
+        - configurable-http-proxy >= 4
         - nodejs >=12
```

Comment:
There is a space here not found below, will this be fine still?

Reply:
It's fine but I've changed it anyway to be consistent (someone should write a pre-commit for conda meta.yaml files 😄)

Reply:
Looks like it does matter.... maybe it's the space before that's optional